.github/workflows/appspot.yaml

```diff
@@ -0,0 +1,39 @@
+name: Deploy to Appspot (one-off)
+
+on:
+  workflow_dispatch:
+    # Allows for manual triggering, to correct deployment issues.
+    inputs:
+      ref:
+        description: "A tag name or SHA1 to deploy."
+        required: true
+      subdomain:
+        description: "An appspot subdomain to deploy to.  Be careful!  Do not overwrite an existing subdomain!"
+        required: true
```

Comment:
Since I'm not as familiar with appspot, where would we find the proper inputs for these fields when running this workflow?

Reply:
For example, https://v4-2-10-dot-shaka-player-demo.appspot.com/ is where v4.2.10 is hosted.  So to replicate that, you would set ref to `v4.2.10` and subdomain to `v4-2-10`.  In a standard release, that is computed automatically from the tag, but in this version, it's manual input to allow us to correct issues like the one we had to day with v4.2.11.  For that, I will run it with ref set to `d8b2378f` (the commit after v4.2.11 that fixes deployment), and subdomain set to `v4-2-11`.